flightlib/include/flightlib/objects/static_gate.hpp

```diff
@@ -5,26 +5,8 @@
 namespace flightlib {
 class StaticGate : public StaticObject {
  public:
-  StaticGate(std::string id, std::string prefab_id);
-  ~StaticGate(){};
-
-  // publich set functions
-  void setPosition(const Vector<3>& position) { position_ = position; };
-  void setRotation(const Quaternion& quaternion) { quat_ = quaternion; };
-  void setSize(const Vector<3>& size) { size_ = size; };
-
-  // publich get functions
-  Vector<3> getPos(void) { return position_; };
-  Quaternion getQuat(void) { return quat_; };
-  Vector<3> getSize(void) { return size_; };
-
- private:
-  std::string id_;
-  std::string prefab_id_;
-
-  Vector<3> position_{0.0, 0.0, 0.0};
-  Quaternion quat_{1.0, 0.0, 0.0, 0.0};
-  Vector<3> size_{1.0, 1.0, 1.0};
+  StaticGate(std::string id) : StaticObject(id, "rpg_gate") {}
```

Comment:
I would prefer to keep the same interface. 

StaticGate(const std::string id, const std::string prefab_id="rpg_gate");

Reply:
No problem! I will change and push right away